browser/js/login/login.html

```diff
@@ -1,16 +1,31 @@
-<form id="login-form" name="loginForm" ng-submit="loginForm.$valid && sendLogin(login)">
+<div class="container">
```

Comment:
Logging in doesn't work when I'm running this branch locally - check if it works on yours?